modules/sovrnBidAdapter.js

```diff
@@ -81,15 +81,12 @@ export const spec = {
         };
       }
 
-      if (bidderRequest && bidderRequest.gdprConsent) {
-        sovrnBidReq.regs = {
-          ext: {
-            gdpr: +bidderRequest.gdprConsent.gdprApplies
-          }};
-        sovrnBidReq.user = {
-          ext: {
-            consent: bidderRequest.gdprConsent.consentString
-          }};
+      if (bidderRequest.gdprConsent) {
```

Comment:
how come you took out this additional check?... just curious.

Reply:
it's already referred to, so if it didn't exist it's going to barf before we get here